Tests/RevenueCatUITests/Data/TemplateViewConfigurationTests.swift

```diff
@@ -266,7 +457,7 @@ private extension BaseTemplateViewConfigurationTests {
         callToAction: "Purchase for {{ price }}",
         callToActionWithIntroOffer: nil,
         offerDetails: "{{ sub_price_per_month }} per month",
-        offerDetailsWithIntroOffer: "Start your {{ sub_offer_duration }} trial, " +
+        offerDetailsWithIntroOffer: "Start your {{ sub_offer_duration }} trial" +
```

Comment:
was this one intentional? it'll look broken. I know it's just in tests so it shouldn't make a difference

Reply:
Oops no, fixed.